components/authz-service/server/v1/server.go

```diff
@@ -53,14 +54,16 @@ func NewMemstoreServer(
 	return NewWithStore(ctx, l, e, mem, true, false, nil)
 }
 
+// NewPostgresServer instantiates a server.Server that connects to a postgres backend
 // NewPostgresServer instantiates a server.Server that connects to a postgres backend
 func NewPostgresServer(
 	ctx context.Context,
 	l logger.Logger,
 	e engine.V1Engine,
-	migrationsConfig migration.Config) (*Server, error) {
+	migrationsConfig migration.Config,
+	datamigrationsConfig datamigration.Config) (*Server, error) {
 
-	p, err := postgres_v1.New(ctx, l, migrationsConfig)
+	p, err := postgres_v1.New(ctx, l, migrationsConfig, datamigrationsConfig)
```

Comment:
We had to modify the signature of the server in order to make postgres manage all migrations (schema, v2-related/datamigrations) so this implementation had to be changed.